lib/unlock-helpers.js

```diff
@@ -117,10 +122,12 @@ helpers.pinUnlock = async function (adb, driver, capabilities) {
       await driver.click(util.unwrapElement(el));
     }
   }
-  const el = await driver.findElOrEls('xpath', "//*[contains(@resource-id, 'id/key_enter')]", false);
-  await driver.click(util.unwrapElement(el));
-  // Waits a bit for the device to be unlocked
+  // Some devices accept the entered key automatically
   await sleep(UNLOCK_WAIT_TIME);
```

Comment:
When I rush commands without this wait before pressKeyCode, rarely UI2 sever crashed.
Thus, I leave this and call `isScreenLocked` after this.

Reply:
Please add this note as the comment in the code